examples/utils/executable_for_notebook_runtimes.py

```diff
@@ -0,0 +1,54 @@
+environment_variables_config = {'ACCOUNT_ID': ACCOUNT_ID,
+                                'AUTH_TOKEN': AUTH_TOKEN,
+                                'MATERIALS_PROJECT_API_KEY': MATERIALS_PROJECT_API_KEY,
+                                'ORGANIZATION_ID': ORGANIZATION_ID}
+
```

Comment:
executable_for_notebook_runtimes.py is a cryptic name - should we call it `initialize_settings.py`?